model/src/channel/followed_channel.rs

```diff
@@ -0,0 +1,39 @@
+use crate::id::{ChannelId, WebhookId};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct FollowedChannel {
```

Comment:
Since we are adding this, as well as adding tests, we should strive to add documentation to it before merge as well. It will help us with #542

Reply:
Done.